src/main/scala/wdlTools/cli/package.scala

```diff
@@ -369,25 +409,4 @@ class WdlToolsConf(args: Seq[String]) extends ScallopConf(args) {
   addSubcommand(readmes)
 
   verify()
-
-  /**
-    * The verbosity level
-    * @return
-    */
-  def verbosity: Verbosity = {
-    if (this.verbose.getOrElse(default = false)) {
```

Comment:
What does the syntax `default = false` mean here? I have never seen it before.

Reply:
Default is just the parameter name. It is not required, but my IDE suggests adding parameter names for boolean parameters to make the code a bit easier to read. For something like getOrElse I guess it's not really necessary.